components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -73,6 +73,7 @@ type OpenshiftNotebookReconciler struct {
 // +kubebuilder:rbac:groups=networking.k8s.io,resources=networkpolicies,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=roles,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=rolebindings,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups="image.openshift.io",resources=imagestreams,verbs=list;get
```

Comment:
i don't understand how this worked before the permissions were added like this; it _did_ work

Reply:
this puzzled me a lot that i could not see why it worked with out permisison.....
maybe @atheo89 knows more?